sites/platform/static/files/fetch/examples/relationships/memcached

```diff
@@ -2,8 +2,11 @@
     "service": "memcached16",
     "ip": "169.254.228.111",
     "hostname": "3sdm72jgaxge2b6aunxdlzxyea.memcached16.service._.eu-3.platformsh.site",
-    "cluster": "rjify4yjcwxaa-master-7rqtwti",
     "host": "memcached.internal",
+    "cluster": "rjify4yjcwxaa-master-7rqtwti",
```

Comment:
please do not expose potential IPs ;) 

for such config, please use: 
- IP: 123.456.78.90
-  "hostname": "azertyuiopqsdfghjklm.memcached16.service._.eu-1.platformsh.site",